kibana.md

```diff
@@ -1278,8 +1324,8 @@ Now try to login as user1, and ReadonlyREST Enterprise should initialize the ind
 
 You can configure the `number_of_shards` and `number_of_replicas` for the tenant index via the `kibana.yml` file, allowing you to override the default index settings. This can be particularly useful in a single-node environment.
 ```yaml
-readonlyrest.tenantIndex.number_of_shards: 1
-readonlyrest.tenantIndex.number_of_replicas: 0
+readonlyrest_kbn.tenantIndex.number_of_shards: 1
```

Comment:
This change was on the master branch, i merged it to the develop. Could you update your branch?